Makefile

```diff
@@ -21,7 +21,8 @@ build:
 	go build $(FLAGS) -o bin/$(BIN_NAME)_v$(VERSION) main.go
 
 test:
-	cd instaclustr && go test -v -timeout 120m -count=1
+	cd instaclustr && go test -v -timeout 120m -count=1 -coverprofile coverage.out -json ./... > report.json
+	@cd instaclustr && cat report.json | sed -n '/Output/p' | jq '.Output' # Prettify the report.json file to print it to stdout
```

Comment:
Doesn't this report just disappear when the CircleCI pipeline is complete?

Reply:
yes, but if we print to stdout the user is able to see the unit test results and in case there's a failure, debug easier.

Reply:
and, this report is used for sonarcloud. not necessarily just for this cat command